bottlecap/src/config/trace_propagation_style.rs

```diff
@@ -11,6 +15,18 @@ pub enum TracePropagationStyle {
     None,
 }
 
+impl TracePropagationStyle {
```

Comment:
Not happy with coupling this into the propagation module which is going to be a crate eventually